treeherder/webapp/api/performance_data.py

```diff
@@ -253,6 +253,11 @@ class AlertPagination(pagination.CursorPagination):
 
     pagination_class = AlertPagination
 
+    def update(self, request, *args, **kwargs):
+        if 'related_summary_id' in request.data:
```

Comment:
Take this line out to unconditionally set the classifier field in the case of an update. :)